R/LinReg.R

```diff
@@ -1,6 +1,6 @@
 LinReg<- function(model, particles=1000, plot=FALSE) {
 
-    data(radiata)
+    data(radiata, envir = environment())
```

Comment:
I have to take some responsibility for this (https://github.com/eddelbuettel/rcppsmc/pull/13#issuecomment-319898169); the writing R extensions docs (https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Data-in-packages) weren't very prescriptive about The Right Way to deal with this and I have no strong views.